### PR TITLE
New version: Unity.Unity.2021 version 2021.3.17f1

### PR DESCRIPTION
**Pull request by vedantmgoyal9**
### Result: Installation Failed
|             | Manifest                 | Add/Remove Programs                   |
| ----------- | ------------------------ | ------------------------------------- |
| Name        | Unity 2021 | AWS SAM Command Line Interface, AWS SAM Command Line Interface    |
| Version     | 2021.3.17f1     | 1.70.0, 1.71.0 |
| Publisher   | Unity Technologies ApS   | AWS Serverless Applications, AWS Serverless Applications      |
| ProductCode |           | {F070941E-E38D-4625-BBA8-EED0E86CC35E}, {55FFEE00-AB8F-4748-8495-C46C03A9DA51}    |
###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/94548)

### CODE REVIEW

File: manifests/u/Unity/Unity/2021/2021.3.17f1/Unity.Unity.2021.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+  - Visual Scripting: Fixed a problem that was preventing link.xml creation when building for Mono backend. ("UVSB-2348")
+  - Visual Scripting: Fixed graphs being corrupted on deserialization if containing a node whose type cannot be found. ("UVSB-2332")
+  - Visual Scripting: Fixed Visual Scripting settings now only save to disk when modified.
+  - Visual Scripting: Fixed Gradient.mode serialization. This fix is available for Unity 2021.3.9f1 or newer. ("UVSB-2356")
+  - Visual Scripting: Moved Events/MessageListeners files to a Listeners folder to avoid to exceed some OS path limit.
+  - WebGL: Fixed a bug where Firefox would hang at 90% while loading. ("UUM-1155")
+  - WebGL: Fixed error handling of AudioContext.resume(). (UUM-17803)
+  - XR: Fixed crash on Quest when using MSAA and moving out of the guardian during startup. (UUM-20525)
+  - XR: Updated XR Interaction Toolkit to 2.2.0.
+
```
Comment (stephengillie) [inline manifests/u/Unity/Unity/2021/2021.3.17f1/Unity.Unity.2021.locale.en-US.yaml:214]:
```suggestion
```

**Comment by hackean-msft** [comment]
@vedantmgoyal2009 Cancelling the underlying build associated with pull request because of ongoing prod deployment.

**Review by ImJoakim — APPROVED** [review]

**Pull request merged**